packages/core/src/actors/index.ts

```diff
@@ -63,3 +69,9 @@ export function toActorRef<
     ...actorRefLike
   };
 }
+
+const nullBehavior = fromTransition((_) => undefined, undefined);
```

Comment:
I see that you took a page from the JavaScript language design :D a null actor that returns undefined? 😉 
```suggestion
const nullBehavior = fromTransition(() => null, null);
```